### PR TITLE
Competitive programming question

### PR DESCRIPTION
**Pull request by anantbharti30**

### CODE REVIEW

File: Task 1/Time conversion/Readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This folder contains the competitive programming question and solution. The question is Time conversion.
```
Comment (chandrikadeb7) [inline Task 1/Time conversion/Readme.md:1]: Delete this file as it is unnececssary

File: Task 1/Time conversion/Question.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Given a time in 12-hour AM/PM format, in a string, convert it to military (24-hour format) time.
```
Comment (chandrikadeb7) [inline Task 1/Time conversion/Question.md:1]: Add headings and subheading in markdown format

File: Task 1/Time conversion/Readme.md (hunk 1 above)
Comment (anantbharti30) [reply Task 1/Time conversion/Readme.md:1]: sure

File: Task 1/Time conversion/Question.md (hunk 2 above)
Comment (anantbharti30) [reply Task 1/Time conversion/Question.md:1]: ok Let me do it

**Comment by anantbharti30** [comment]
Task completed ! Please confirm if any more changes are needed.

**Comment by anantbharti30** [comment]
Hey @chandrikadeb7  Can you please accept my contribution.  I am participating in Hacktoberfest2020. Please merge this file with hacktoberfest-accepted label.

**Review by chandrikadeb7 — APPROVED** [review]
Looks good!

**Pull request merged**